### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by JefZheng**
WHAT DOES THIS PR DO:

Added content to our readme file

ADDITIONAL NOTES:

If you want to add/change a profile picture, please do so in the contributor section. I used everyone's default github profile picture but you can very easily change it to any source you wish in the  tag

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,109 @@
...
+![GitHub release (latest by date including pre-releases)](https://img.shields.io/github/v/release/oslabs-beta/Kubiquity?include_prereleases)
+![GitHub pull requests](https://img.shields.io/github/issues-pr/oslabs-beta/kubiquity)
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/oslabs-beta/Kubiquity#readme)
+[![License: MIT](https://img.shields.io/github/license/oslabs-beta/Kubiquity)](https://github.com/oslabs-beta/Kubiquity/blob/master/LICENSE)
+[![Twitter: kubiquityapp](https://img.shields.io/twitter/follow/kubiquityapp.svg?style=social)](https://twitter.com/kubiquityapp)
 
+
+### 🏠 [Homepage](https://kubiquity)
+
+>Kubiquity is a real time Kubernetes error monitoring tool that incorporates Prometheus metrics queries for CPU and memory usage that runs locally as an Electron application.
```
Comment (dlande000) [inline README.md]: Suggested edit: 

>Kubiquity is a real-time Kubernetes error-monitoring tool. Kubiquity runs locally as an Electron application and incorporates Prometheus metrics queries for a cluster's CPU and memory usage by node.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,109 @@
...
+
+# Installation
+[(Back to top)](#table-of-contents)
+
+Visit our [releases page](https://github.com/oslabs-beta/Kubiquity/releases) to download the latest stable release for your operating system
+
+To package the app from the source code, fork and clone the repo, then run the following commands
+
+1. ```yarn```
+2. ```yarn run package-linux``` or ```yarn run package-win``` based on your OS"
```
Comment (dlande000) [inline README.md]: Remove `"` at end of line.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,109 @@
...
+- [Usage](#usage)
+- [TODO](#todo)
+- [Contributions](#contributions)
+- [Contributors](#contributors)
+- [License](#license)
+
+# Installation
+[(Back to top)](#table-of-contents)
+
+Visit our [releases page](https://github.com/oslabs-beta/Kubiquity/releases) to download the latest stable release for your operating system
```
Comment (dlande000) [inline README.md]: End sentence with period.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,109 @@
...
+
+To package the app from the source code, fork and clone the repo, then run the following commands
```
Comment (dlande000) [inline README.md]: End sentence with period.

File: README.md (hunk 2 above)
Comment (dlande000) [reply README.md]: End sentence with period.

File: README.md [hunk 5]
```diff
@@ -1,3 +1,109 @@
...
+
+1. ```yarn```
+2. ```yarn run package-linux``` or ```yarn run package-win``` based on your OS"
+3. The app will be created and placed in the ```release-builds``` folder
```
Comment (dlande000) [inline README.md]: End sentence with period.

File: README.md [hunk 6]
```diff
@@ -1,3 +1,109 @@
...
+
+# Usage
+[(Back to top)](#table-of-contents)
+
+- Ensure that a Kubernetes cluster is running.
```
Comment (dlande000) [inline README.md:50]: Proposed edit: 

- Ensure that a Kubernetes cluster is running and healthy.

File: README.md [hunk 7]
```diff
@@ -1,3 +1,109 @@
...
+
+- Create a Prometheus namespace in the cluster and install Prometheus into that namespace.
+
+- When running the app, ensure that Prometheus has been running for at least a minute so that it can generate metrics from the cluster.
+
+
+# TODO
```
Comment (dlande000) [inline README.md]: Future iterations or future implementations or planned implementations for header name, not todo.

File: README.md [hunk 8]
```diff
@@ -1,3 +1,109 @@
...
+
+Visit our [releases page](https://github.com/oslabs-beta/Kubiquity/releases) to download the latest stable release for your operating system
+
+To package the app from the source code, fork and clone the repo, then run the following commands
+
+1. ```yarn```
+2. ```yarn run package-linux``` or ```yarn run package-win``` based on your OS"
+3. The app will be created and placed in the ```release-builds``` folder
+
+# Usage
```
Comment (dlande000) [inline README.md]: Change to Development requirements or something

File: README.md [hunk 9]
```diff
@@ -1,3 +1,109 @@
...
+
+- When running the app, ensure that Prometheus has been running for at least a minute so that it can generate metrics from the cluster.
+
+
+# TODO
+[(Back to top)](#table-of-contents)
+
+- [ ] Add Mac OS support
+- [ ] Add time travel debugging by saving a snapshot of the current state of the cluster
+- [ ] Add line graph for CPU and memory usage
```
Comment (dlande000) [inline README.md]: Change to 
Store and display CPU and memory usage over time.

File: README.md [hunk 10]
```diff
@@ -1,3 +1,109 @@
...
+- [ ] Add recommended course of action based on warnings and errors
```
Comment (dlande000) [inline README.md]: 60–63: sentences terminate in periods.

File: README.md [hunk 11]
```diff
@@ -1,3 +1,109 @@
...
+
+
+# License
+[(Back to top)](#table-of-contents)
+
+Copyright © 2021 [k8sm8s](https://github.com/oslabs-beta).
+
+This product is [MIT](https://github.com/oslabs-beta/Kubiquity/blob/master/LICENSE) licensed.
+
+This product is accelerated by [OS Labs](https://opensourcelabs.io)
```
Comment (dlande000) [inline README.md]: Add period to end of sentence.

**Review by dlande000 — APPROVED** [review]
Approved pending resolution of comments.

**Review by Hydroelectric29 — APPROVED** [review]

**Review by davidzhangnyc — APPROVED** [review]

**Pull request merged**